test/images/agnhost/connect/connect.go

```diff
@@ -42,19 +43,25 @@ var CmdConnect = &cobra.Command{
 	Run:  main,
 }
 
-var timeout time.Duration
-var protocol string
+var (
+	timeout  time.Duration
+	protocol string
+	udpData  string
+)
 
 func init() {
 	CmdConnect.Flags().DurationVar(&timeout, "timeout", time.Duration(0), "Maximum time before returning an error")
-	CmdConnect.Flags().StringVar(&protocol, "protocol", "tcp", "The protocol to use to perform the connection, can be tcp or sctp")
+	CmdConnect.Flags().StringVar(&protocol, "protocol", "tcp", "The protocol to use to perform the connection, can be tcp, udp or sctp")
+	CmdConnect.Flags().StringVar(&udpData, "udp-data", "", "The UDP payload send to the server")
```

Comment:
we should put some data as default, "hello" or "hostname" can be valid since are used in our tests in a number of cases, I think `hostname` is used in more places